src/ecmult_impl.h

```diff
@@ -1089,19 +1164,17 @@ static size_t secp256k1_pippenger_max_points(const secp256k1_callback* error_cal
         size_t n_points;
         size_t max_points = secp256k1_pippenger_bucket_window_inv(bucket_window);
         size_t space_for_points;
-        size_t space_overhead;
-        size_t entry_size = sizeof(secp256k1_ge) + sizeof(secp256k1_scalar) + sizeof(struct secp256k1_pippenger_point_state) + (WNAF_SIZE(bucket_window+1)+1)*sizeof(int);
+        size_t space_constant;
+        /* Compute entry size without taking alignment into account */
+        size_t entry_size = secp256k1_pippenger_scratch_size_points(0, bucket_window, 0);
 
-#ifdef USE_ENDOMORPHISM
-        entry_size = 2*entry_size;
-#endif
-        space_overhead = (sizeof(secp256k1_gej) << bucket_window) + entry_size + sizeof(struct secp256k1_pippenger_state);
-        if (space_overhead > max_alloc) {
+        space_constant = secp256k1_pippenger_scratch_size_constant(bucket_window);
+        if (space_constant + entry_size > max_alloc) {
             break;
         }
-        space_for_points = max_alloc - space_overhead;
+        space_for_points = max_alloc - space_constant;
 
-        n_points = space_for_points/entry_size;
+        n_points = (space_for_points - entry_size)/entry_size;
```

Comment:
Is this right? It's equivalent to `space_for_points / entry_size - 1`.

Reply:
It is right. Simplified the line according to your suggestion.

Reply:
Is this assignment to `n_points` (along with the comment) redundant with above?